saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/AssertingPartyMetadata.java

```diff
@@ -28,7 +29,7 @@
  * @author Josh Cummings
  * @since 6.4
  */
-public interface AssertingPartyMetadata {
+public interface AssertingPartyMetadata extends Serializable {
```

Comment:
Please add a `serialVersionUID` to `AssertingPartyDetails`, an implementation of `AssertingPartyMetadata`.